manager/controlapi/network.go

```diff
@@ -112,6 +114,13 @@ func (s *Server) CreateNetwork(ctx context.Context, request *api.CreateNetworkRe
 	}
 
 	err := s.store.Update(func(tx store.Tx) error {
+		if request.Spec.Ingress {
```

Comment:
Not sure if it is appropriate to percolate the ingress notion down into the `store.createNetwork()`.
It is true it is doing now a check on duplicated network name, though, but that is more generic, top level resource name.
If you guys think it makes sense, I can move it there now.